Fix bug when trying to modify the entry when an option is selected in AutocompleteInput (#1179)
vtex/styleguide: CHANGELOG.md

### Fixed

> - Update input value when I try to update with an option selected in the `AutocompleteInput` component

Reviewer: ```suggestion
- Value when user tries to update with an option selected in the `AutocompleteInput` component.
```

> 
> ### Fixed

Reviewer 2: ```suggestion
```

> ### Fixed

Reviewer: ```suggestion
```

Reviewer 2: @jardelymaris can you apply this suggests?

### Fixed

- Value when user tries to update with an option selected in the `AutocompleteInput` component

> 

Reviewer 2: ```suggestion
```
